example/reinforcement-learning/a3c/a3c.py

```diff
@@ -153,8 +153,11 @@ def train():
                 module.forward(batch, is_train=True)
 
                 pi = module.get_outputs()[1]
-                h = args.beta*(mx.nd.log(pi+1e-6)+1)
-                module.backward([mx.nd.array(adv), h])
+                h = -args.beta*(mx.nd.log(pi+1e-7)*pi)
+                out_acts = np.amax(pi.asnumpy(), 1)
```

Comment:
Here 
![image](https://cloud.githubusercontent.com/assets/5026576/24078220/e56b536e-0ca2-11e7-8c69-1052a15dbf79.png) means the probability to act a_t in state s, i think. it means if the act is right, the probability should be big,so the loss should be low,vice versa.